### PR TITLE
Gonzo release note changes

### PR DESCRIPTION
**Pull request by kmercierwhite**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Release note changes for Gonzo

## Deploy preview link

https://deploy-preview-2407--axway-open-docs.netlify.app/docs/apim_relnotes/20220830_apimgr_relnotes/

_Although Netlify will add this link below, it will redirect to the main page of the documentation, not to an specific page._

## Checklist for contributors

Before submitting this PR, please make sure:

_This is simply a reminder of what we are going to look for before merging your change._

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [ ] You have verified that all status checks have passed

### CODE REVIEW

**Review by jmbermingham — APPROVED** [review]
Looks good, but only one small change - can we _not_ use the EMT terminology and use Container instead?
i.e. "This feature enhancement allows a Node Manager deployed in container mode to be reconfigured"

File: content/en/docs/apim_relnotes/20220830_apimgr_relnotes.md [hunk 1]
```diff
@@ -54,6 +54,42 @@ Customer scripts or client applications might now fail to get APIs from other or
 
 For more information, see [API Manager access control, Organization administrator](/docs/api_mgmt_overview/key_concepts/api_mgmt_orgs_roles/index.html#organizationadministrator).
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20220830_apimgr_relnotes.md:56]: Also need to add a session around "YAML and Policy Studio" (better call it 'Case sensitivity in YAML projects' ??). Add text, and link to documetation (see, Case sensitivity in YAML projects, /docs/apim_yamles/apim_yamles_references/yamles_known_limitations/#api-gateway-manager-web-ui)

Relates to #2405 and #2401

**Comment by kmercierwhite** [comment]
> Looks good, but only one small change - can we _not_ use the EMT terminology and use Container instead? i.e. "This feature enhancement allows a Node Manager deployed in container mode to be reconfigured"

Yes sorry, we had discussed using Container instead of EMT. @andreamussap should I make the change or will you?

**Comment by andreamussap** [comment]
> > Looks good, but only one small change - can we _not_ use the EMT terminology and use Container instead? i.e. "This feature enhancement allows a Node Manager deployed in container mode to be reconfigured"
> 
> Yes sorry, we had discussed using Container instead of EMT. @andreamussap should I make the change or will you?

@kmercierwhite go ahead and do it. Thanks

**Comment by kmercierwhite** [comment]
> > > Looks good, but only one small change - can we _not_ use the EMT terminology and use Container instead? i.e. "This feature enhancement allows a Node Manager deployed in container mode to be reconfigured"
> > 
> > 
> > Yes sorry, we had discussed using Container instead of EMT. @andreamussap should I make the change or will you?
> 
> @kmercierwhite go ahead and do it. Thanks

Thanks, done!

**Pull request merged**